lib/DrupalHelper.php

```diff
@@ -19,7 +19,13 @@ class DrupalHelper
     public function bootDrupal($drupalRoot)
     {
         $autoloader = require_once $drupalRoot . '/autoload.php';
-        $request = new Request();
+        // Inherit the current request's HTTP host to respect trusted hosts
+        // settings.
+        $current_request = Request::createFromGlobals();
+        $server = [
+          'HTTP_HOST' => $current_request->getHost(),
+        ];
+        $request = new Request([], [], [], [], [], $server);
```

Comment:
I was not sure whether it was more sensible to just pass along the current request, so decided to go this way and minimize the actual behavior changes to avoid regressions.

Reply:
Thanks. This is probably a better approach then just passing current request. Will see in  future.